girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -449,56 +449,100 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
 
     @access.user(scope=TokenScope.DATA_WRITE)
     @autoDescribeRoute(
-        Description('Create new large images for all items within a folder.')
-        .notes('Does not work for items with multiple files and skips over items with '
-               'existing or unfinished large images.')
-        .modelParam('id', 'The ID of the folder.', model=Folder, level=AccessType.WRITE,
-                    required=True)
-        .param('force', 'Whether creation job(s) should be forced for each large image.',
+        Description('Create large images for all items within a folder.')
+        .notes('Does not work for items with multiple files.')
+        .modelParam('id', 'The ID of the folder.', model=Folder,
+                    level=AccessType.WRITE, required=True)
+        .param('createJobs', 'Whether job(s) should be used to create each '
+               'large image. Select true to use a job if needed, false to '
+               'never use a job, and always to always use a job.',
+               required=False, default='false', dataType='string',
+               enum=['true', 'false', 'always'])
+        .param('localJobs', 'Whether the job(s) created should be local.',
                required=False, default=False, dataType='boolean')
-        .param('localJobs', 'Whether the job(s) created should be local.', required=False,
-               default=False, dataType='boolean')
-        .param('recurse', 'Whether child folders should be recursed.', required=False,
-               default=False, dataType='boolean')
+        .param('recurse', 'Whether child folders should be recursed.',
```

Comment:
I generally prefer definitions not to include the option unless _very_ self-explanatory. Perhaps something like: 
'If true, images in child folders will also be checked'

Also as a personal preference I would change the descriptions from 'Whether...' to 'If...'

Reply:
That reads clearer and explaining what happens depending on what the user selects is a good idea. Change made [here](https://github.com/girder/large_image/commit/e22df9496ed745dc055801bb6328d64c75afcc15).